dynamiqs/mesolve/rouchon.py

```diff
@@ -80,11 +84,17 @@ def forward(self, t: float, rho: Tensor) -> Tensor:
         Hnh = self.Hnh(H)
         M0 = self.M0(Hnh)
 
+        # normalize the Kraus Map
+        if self.cholesky_normalization:
+            # R, T: (b_H, 1, n, n)
+            R = self.R(M0)
+            T = self.T(R)
```

Comment:
These two lines require caching two objects, whereas only `T` is used in the end. Not sure how important this is, I'll let you decide whether its worth correcting.

Reply:
It's not 100% ideal but I'd prefer to keep it like that for now, it makes the code clearer, and we're also going to need this `R` matrix for the SME, the solvers will look more similar. I think we can refine performance later.